repo/quicksilver/package.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Quicksilver(MakefilePackage):
+    """Quicksilver is a proxy application that represents some elements of the
+    Mercury workload.
+    """
+
+    tags = ["proxy-app"]
+
+    homepage = "https://codesign.llnl.gov/quicksilver.php"
+    url = "https://github.com/LLNL/Quicksilver/tarball/V1.0"
+    git = "https://github.com/LLNL/Quicksilver.git"
+
+    maintainers("richards12")
+
+    version("master", branch="master")
+    version("1.0", sha256="83371603b169ec75e41fb358881b7bd498e83597cd251ff9e5c35769ef22c59a")
+
+    variant("openmp", default=True, description="Build with OpenMP support")
```

Comment:
@august-knox Set the default openmp variant to be False. We will need to support other modes like (+cuda, +rocm etc)